src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/BaseDataDonationViewModel.swift

```diff
@@ -0,0 +1,104 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+import OpenCombine
+
+/*** this protocol gets used to provide different view model classes for the dataDonationViewController */
+
+protocol DataDonationViewModelProtocol {
+
+	/// use these 3 to definde @published in a prorocol
```

Comment:
👀 don't understand this comment 🙈

Reply:
The whole thing is a workaround, for not being able to use property wrapper in protocols.
https://swiftsenpai.com/swift/define-protocol-with-published-property-wrapper/

Maybe the comment can be improved, which should then also explain this: https://github.com/corona-warn-app/cwa-app-ios/pull/1977#discussion_r575330743

Reply:
i changed the whole thing a bit so the useless bool is gone